src/layer/tile/TileLayer.js

```diff
@@ -197,12 +197,7 @@ export var TileLayer = GridLayer.extend({
 	},
 
 	_tileOnLoad: function (done, tile) {
-		// For https://github.com/Leaflet/Leaflet/issues/3332
-		if (Browser.ielt9) {

```

Comment:
Removed this condition as we no longer support any version of Internet Explorer.